build/vsts-ci.yml

```diff
@@ -50,11 +50,9 @@ phases:
   queue:
     name: Hosted macOS
   steps:
+  # Work around MacOS Homebrew image/environment bug: https://github.com/actions/virtual-environments/issues/2322#issuecomment-749211076
   - script: |
-      brew uninstall openssl@1.0.2t |
-      brew uninstall python@2.7.17 |
-      brew untap local/openssl |
-      brew untap local/python2
+      rm -rf /usr/local/bin/2to3
```

Comment:
>       [](start = 0, length = 6)

does this script always run with the permissions allow deleting such folder?

Reply:
From my understanding yes. This workaround was actually suggested by the team who manages the hosted MacOS queues. See [here](https://github.com/actions/virtual-environments/issues/2322#issuecomment-749211076) for more info.